lib/workers/repository/update/pr/code-owners.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { codeBlock } from 'common-tags';
```

Comment:
The maintainers should check if this PR (partially) fixes this issue:

- #12611

Reply:
I believe that this PR will fix the issue, as rules are now matched against each file and not against the entire list of modified files as it was done before